api/api.proto

```diff
@@ -937,6 +937,28 @@ service OpenStorageCluster {
     }
 }
 
+// OpenStorageClusterPair service provides the methods to manage a cluster pair
+service OpenStorageClusterPair{
+  // Creates Pair with a remote cluster and returns details about the remote cluster
+  //
+  // ##### Example
+  // {% codetabs name="Golang", type="go" -%}
+  // id, err := client.Create(context.Background(), &api.SdkVolumeCreateRequest{
+  //   RemoteClusterIp: "127.0.0.1",
+  //   RemoteClusterPort: 12345,
+  //   RemoteClusterToken: "<Auth-Token>",
+  //   SetDefault: <Boolean-Value>,
+  // })
+  // {%- endcodetabs %}
+  rpc Create(ClusterPairCreateRequest)
```

Comment:
[See the requirements](https://github.com/libopenstorage/openstorage/blob/master/docs/dev-sdk.md#sdk-requirements) All Sdk messages (arguments) should start with `SdkXXX`

Reply:
So it looks like you probably should have something like this due to the current already defined messages:

```
message SdkClusterPairCreateRequest {
    ClusterPairCreateRequest req = 1;
}
```

Reply:
I tried changing this to the SdkClusterPairRequest and Response, but that requires change in the interface, clients, and cluster.go files, to make everything adhere to this parameter in the create call.